packages/admin-next/dashboard/src/hooks/api/collections.tsx

```diff
@@ -90,19 +90,24 @@ export const useUpdateCollection = (
 export const useUpdateCollectionProducts = (
   id: string,
   options?: UseMutationOptions<
-    ProductCollectionRes,
-    Error,
-    UpdateProductCollectionProductsReq
+    { collection: HttpTypes.AdminCollection },
+    FetchError,
+    AdminUpdateCollectionProducts
```

Comment:
suggestion: should we take all the types from the HttpTypes to stay coherent through the usage?